web_src/js/features/load-branches-and-tags.js

```diff
@@ -0,0 +1,64 @@
+const {csrfToken} = window.config;
+
+async function loadBranchesAndTags(loadingButton, addHere) {
+  loadingButton.setAttribute('disabled', 'disabled');
```

Comment:
`.disabled`

Reply:
I mean, I can use the class, but what's the benefit compared to the browser native mechanism?

Reply:
According to MDN, the "disabled" attribute doesn't need to have "value", it is `Boolean attribute`.  https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

Actually any value will make the attribute be "true" value, as long as the attribute is present.

While `el.disabled = true` could be more readable than `el.setAttribute("disabled", "");` which equals to `el.setAttribute("disabled", "disabled");`

Reply:
`.disabled` is definitely more readable and avoid the insanity of boolean HTML attributes.